### DISCUSSION TITLE
States in Germany

### DISCUSSION THREAD

**Discussion by ExtraMint** [vacanza/holidays#660] [Q&A] [answered]
Hello!

I just noticed that there is a total number of 17 states for germany. Could anyone help me in finding out what "BYP" stands for?  

`Germany | DE | States: BB, BE, BW, BY, BYP, HB, HE, HH, MV, NI, NW, RP, SH, SL, SN, ST, TH`

There are just 16 states in germany, see [here](https://www.datenportal.bmbf.de/portal/de/G122.html), so I assume it may be a mistake?

Best regards

**Comment by dragoon** [comment, marked as answer]
From the source code:

```
"Mariä Himmelfahrt" is only a holiday in Bavaria (BY) if your
municipality is mostly catholic which in term depends on census data.
Since we don't have this data but most municipalities in Bavaria
*are* mostly catholic, we count that as holiday for whole Bavaria.
We added BYP for the municipality in Bavaria with more protestants.
Here this is excluded.
```

**Comment by dr-prodigy** [comment]
Thank you @dragoon for sharing this! :+1:
